modules/oe_whitelabel_starter_person/config/install/core.entity_view_display.node.oe_sc_person.oe_w_content_banner.yml

```diff
@@ -0,0 +1,81 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.oe_w_content_banner
+    - field.field.node.oe_sc_person.oe_sc_person_additional_info
+    - field.field.node.oe_sc_person.oe_sc_person_country
+    - field.field.node.oe_sc_person.oe_sc_person_documents
+    - field.field.node.oe_sc_person.oe_sc_person_first_name
+    - field.field.node.oe_sc_person.oe_sc_person_image
+    - field.field.node.oe_sc_person.oe_sc_person_last_name
+    - field.field.node.oe_sc_person.oe_sc_person_occupation
+    - field.field.node.oe_sc_person.oe_sc_person_position
+    - field.field.node.oe_sc_person.oe_social_media_links
+    - field.field.node.oe_sc_person.oe_summary
+    - node.type.oe_sc_person
+  module:
+    - address
+    - text
+    - typed_link
+    - user
+id: node.oe_sc_person.oe_w_content_banner
+targetEntityType: node
+bundle: oe_sc_person
+mode: oe_w_content_banner
+content:
+  oe_sc_person_country:
+    type: address_country_default
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 1
+    region: content
+  oe_sc_person_occupation:
+    type: string
+    label: hidden
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+    weight: 2
+    region: content
+  oe_sc_person_position:
+    type: string
+    label: hidden
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+    weight: 3
+    region: content
+  oe_social_media_links:
+    type: typed_link
+    label: hidden
+    settings:
+      trim_length: 80
+      url_only: false
+      url_plain: false
+      rel: ''
+      target: ''
+    third_party_settings: {  }
+    weight: 4
+    region: content
```

Comment:
```suggestion
```
Seems they are not used in the template, and pulled from the node instead.

Reply:
You're right. Done